modules/apps/transfer/keeper/migrations_test.go

```diff
@@ -3,6 +3,7 @@ package keeper_test
 import (
 	"fmt"
 
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	transferkeeper "github.com/cosmos/ibc-go/v7/modules/apps/transfer/keeper"
 	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
```

Comment:
nit: import separation

```suggestion
	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
	
	transferkeeper "github.com/cosmos/ibc-go/v7/modules/apps/transfer/keeper"
	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
```